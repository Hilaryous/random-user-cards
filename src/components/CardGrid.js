// @flow
import styled from 'react-emotion'

const CardGrid = styled('ul')({
  display: 'grid',
  gridTemplateColumns: 'repeat(auto-fit, 350px)',
  gridGap: 20,
  padding: 0,
  justifyContent: 'center',
})

export default CardGrid
