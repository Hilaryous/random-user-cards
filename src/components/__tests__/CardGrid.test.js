/* eslint-disable function-paren-newline */
import React from 'react'
import renderer from 'react-test-renderer'
import CardGrid from '../CardGrid'

describe('CardGrid', () => {
  it('shows the correct snapshot tree for the CardGrid component', () => {
    const tree = renderer.create(
      <CardGrid />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
