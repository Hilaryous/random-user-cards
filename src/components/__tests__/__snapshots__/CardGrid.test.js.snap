// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardGrid shows the correct snapshot tree for the CardGrid component 1`] = `
<ul
  className="css-1oxkwm3"
/>
`;
